### PR TITLE
Update ocr-learn-about.md

### PR DESCRIPTION
**Pull request by pratjain523**

### CODE REVIEW

**Review by kmkoenen — DISMISSED** [review]
Changes required:

Lines 60 - 64, please use the following text:

[!NOTE]
To reduce your OCR costs, images sent in email via Microsoft Exchange are scanned and billed only once per unique image across all users of the tenant. For all subsequent instances, the results of the previous scan will be reused. Furthermore, there are no additional charges, regardless of whether or not the classification results are used by multiple policies across data loss prevention, insider risk management, auto-labeling, or records management.


Line 114 as follows:
> Only stand-alone images are scanned. Scanning images embedded inside Microsoft Office files (.docx, .xlsx, .pptx) and compressed/archive files (.zip, .rar, etc.) isn't supported at this point.

**Comment by pratjain523** [comment]
@kmkoenen We will avoid rescan only for smaller images like logo and signatures. The rephrased sentence is not accurate from product point of view.

**Comment by kmkoenen** [comment]
Please update content:

Changes required:

Lines 60 - 64, please use the following text:

> [!NOTE]
> - To reduce your OCR costs, small images, such as logos and signatures that are sent in email via Microsoft Exchange are scanned and billed only once per unique image across all users of the tenant. For all subsequent instances, the results of the previous scan will be reused. 
> 
> - With respect to large images, although they are rescanned, there are no additional charges with respect to the number of classification results across multiple policies, including data loss prevention, insider risk management, auto-labeling, and records management.


Line 114 as follows:
> Only stand-alone images are scanned. Scanning images embedded inside Microsoft Office files (.docx, .xlsx, .pptx) and compressed/archive files (.zip, .rar, etc.) isn't supported at this point.

**Comment by pratjain523** [comment]
> [!NOTE]
> - To reduce your OCR costs, small images, such as logos and signatures that are sent in email via Microsoft Exchange are scanned and billed only once per unique image across all users of the tenant. For all subsequent instances, the results of the previous scan will be reused. 
> 
> - Every image is charged only once, irrespective of the number of policies across data loss prevention, insider risk management, auto-labeling, and records management.

 


Line 114 as follows:
> Only stand-alone images are scanned. Scanning images embedded inside Microsoft Office files (.docx, .xlsx, .pptx) and compressed/archive files (.zip, .rar, etc.) isn't supported at this point.

File: microsoft-365/compliance/ocr-learn-about.md [hunk 1]
```diff
@@ -57,6 +57,11 @@ The charge for using OCR is $1.00 for every 1,000 items scanned. Each image scan
 
 To view your bill, follow the instructions described in [Monitor your Microsoft Syntex pay-as-you-go usage](/microsoft-365/syntex/syntex-azure-billing#monitor-your-microsoft-syntex-pay-as-you-go-usage).
 
+> [!NOTE]
+> - To reduce your OCR costs, small images, such as logos and signatures that are sent in email via Microsoft Exchange are scanned and billed only once per unique image across all users of the tenant. For all subsequent instances, the results of the previous scan will be reused.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ocr-learn-about.md:64]:
```suggestion
> - To reduce your OCR costs, small images, such as logos and signatures that are sent in email via Microsoft Exchange are scanned and billed only once per unique image across all users of the organization. For all subsequent instances, the results of the previous scan will be reused.
```

File: microsoft-365/compliance/ocr-learn-about.md [hunk 2]
```diff
@@ -106,15 +111,13 @@ This functionality supports scanning images in the following file types, with th
 
 > [!IMPORTANT]
 >
-> - Only images with machine-typed text are supported.
-> - Only images uploaded after OCR has been enabled are scanned.
-> - Only stand-alone images are scanned.
+> - Only stand-alone images are scanned. Scanning images embedded inside Microsoft Office files (.docx, .xlsx, .pptx) and compressed/archive files (.zip, .rar, etc.) isn't supported at this point.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ocr-learn-about.md:117]:
```suggestion
> - Only stand-alone images are scanned. Scanning images embedded inside Microsoft Office files (.docx, .xlsx, .pptx) and compressed or archive files (such as .zip and .rar) isn't supported at this point.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pratjain523, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@kmkoenen Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@pratjain523 Thanks for your feedback! New issues and pull requests for this content are closed while we update this content to support upcoming feature improvements.

**Pull request closed without merging**